edgelet/edgelet-docker/src/notary.rs

```diff
@@ -0,0 +1,190 @@
+use crate::{Error, ErrorKind};
```

Comment:
Order of imports goes like:

1. `std::*`
1. Third-party crates
1. Our crates.
1. `crate::*`

with one empty line separating each category.